api/posts/postsSchemas.js

```diff
@@ -16,13 +24,31 @@ const postSchemaCommonKeys = {
 
   viewsCount: Joi.number().integer().min(0).default(0),
 
-  imageUrl: Joi.string().regex(URL_REGEXP).default("")
+  imageUrl: Joi.string().regex(URL_REGEXP).default(""),
+
+  isDeleted: Joi.boolean().default(false)
 };
 
 const newPostSchema = {
   body: Joi.object().keys(Object.assign({}, postSchemaCommonKeys))
 };
 
+const allPostsSchema = {
+  query: Joi.object().keys({
+    page: Joi.string().alphanum().trim().default("1"),
+    perPage: Joi.number().integer().min(1).max(100).default(3),
+    sortBy: Joi.string()
+      .valid(ID, TITLE, TEXT, VIEWSCOUNT, DATE)
```

Comment:
Why uppercase?

Reply:
You should be able to sort by two params. I would organize something like https://www.moesif.com/blog/technical/api-design/REST-API-Design-Filtering-Sorting-and-Pagination/#example-formats. 

When you can specify more than one param